connectors/google-sheets/element-templates/google-sheets-connector.json

```diff
@@ -0,0 +1,586 @@
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.worksheetName"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createWorksheet",
+          "worksheetData"
+        ]
+      }
+    },
+    {
+      "label": "Worksheet name",
+      "description": "Enter the name of the worksheet",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.worksheetName"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "getRowByIndex",
+          "createRow",
+          "addValues"
+        ]
+      }
+    },
+    {
+      "label": "Dimension",
+      "description": "Choose what to add: column or row",
+      "group": "operationDetails",
+      "type": "Dropdown",
+      "optional": false,
+      "choices": [
+        {
+          "name": "Column",
+          "value": "COLUMNS"
+        },
+        {
+          "name": "Row",
+          "value": "ROWS"
+        }
+      ],
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.dimension"
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "addEmptyColumnOrRow"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Row index",
+      "description": "Enter row index",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must be a number."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "getRowByIndex"
+      }
+    },
+    {
+      "label": "Row index",
+      "description": "Enter the index of the row to add",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.index"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100,
+        "pattern": {
+          "value": "^([0-9]*$)|(secrets.*$)",
+          "message": "Must contains only numbers."
+        }
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "createRow"
+      }
+    },
+    {
+      "label": "Column index",
+      "description": "Enter the index of the column",
```

Comment:
can you add example here

Reply:
As we've decided It will be done with adding the documentation

Reply:
<img width="373" alt="image" src="https://github.com/camunda/connectors-bundle/assets/131355515/c96417e0-8e15-4da1-8dfe-e611b43be5ba">
Added link